sdk/spring-sdk/src/main/scala/kalix/springsdk/KalixClient.scala

```diff
@@ -151,12 +229,10 @@ trait KalixClient {
    *   The HTTP body type expected by the target endpoint
    * @param returnType
    *   The type returned by the target endpoint
-   * @tparam P
-   *   Type used as a body for the request
    * @tparam R
    *   Type returned as response from the target endpoint
    * @return
    *   a [[kalix.javasdk.DeferredCall]] to be used in forwards and timers or to be executed in place
    */
-  def delete[P, R](uri: String, body: P, returnType: Class[R]): DeferredCall[Any, R]
+  def delete[R](uri: String, returnType: Class[R]): DeferredCall[Any, R]
```

Comment:
@efgpinto, I removed the body for delete just to follow the WebClient choices. 

We can add it later if eventually, but happy to bring it back it there is a strong reason for it.